.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+
+script:
+  - npm run lint
+  - npm run test
+
+node_js:
+  - 12
```

Comment:
If we are 100% sure that this will run as a microservice, than 12 is fine. Otherwise, use 10.

Reply:
I think part of the goal here is to bind our hands to ensure that we will not deploy it alongside the existing stack!